pandas_gbq/load.py

```diff
@@ -94,8 +94,13 @@ def cast_dataframe_for_parquet(
             # .astype() with DateDtype. With .astype(), I get the error:
             #
             # TypeError: Cannot interpret '<db_dtypes.DateDtype ...>' as a data type
-            cast_column = pandas.Series(
```

Comment:
Shouldn't the comment above be removed now?

Reply:
Yes. Good call.